attributes/default.rb

```diff
@@ -53,7 +53,7 @@
 default['mongodb']['dbconfig_file']['mongos'] = '/etc/mongos.conf'
 
 default['mongodb']['package_name'] = 'mongodb'
-default['mongodb']['package_version'] = '3.2.10'
+default['mongodb']['package_version'] = '3.2.18'
```

Comment:
@thebmw Since you're bumping the default mongo version here, that should be added as a separate changelog entry similar to default version bumps from past releases.  Might as well take the time to re-base / merge this with master to fix the conflict